### PR TITLE
Add setting to force install from MIRROR_HTTP on PXE boot

### PR DESCRIPTION
**Pull request by alvarocarvajald**
This commit introduces the setting **FORCE_INSTALL_FROM_MIRROR_HTTP**, which when set to true forces `boot/boot_from_pxe` to install the SUT from the URL contained in **MIRROR_HTTP** (grub boot `install=` argument) even when running in workers with a closer mirror.

Idea behind this is to allow workers to specify a combination of **PXE_PRODUCT_NAME** and **MIRROR_HTTP** settings which will lead to a proper installation of a product/build under test.

- Related ticket: https://progress.opensuse.org/issues/168811
- Needles: https://gitlab.suse.de/openqa/os-autoinst-needles-sles/-/commit/c8513bdf1ef36c7eb3f58c21c0de8b8b1de0f739
- Verification run: https://openqa.suse.de/tests/15963697 :green_circle: , https://openqa.suse.de/tests/15963698 :red_circle: (installation OK, failed later in unrelated module), https://openqa.suse.de/tests/15976699 :green_circle: (but required a change in `wizard_hana_install` test module)

### CODE REVIEW

File: tests/boot/boot_from_pxe.pm [hunk 1]
```diff
@@ -126,7 +126,9 @@ sub run {
             $image_name = get_var('PXE_PRODUCT_NAME') if get_var('PXE_PRODUCT_NAME');
             $image_path = "/mounts/dist/install/SLP/${image_name}/${arch}/DVD1/boot/${arch}/loader/linux ";
             $image_path .= "initrd=/mounts/dist/install/SLP/${image_name}/${arch}/DVD1/boot/${arch}/loader/initrd ";
-            $image_path .= "install=http://mirror.suse.cz/install/SLP/${image_name}/${arch}/DVD1$device ";
+            my $install_path = "install=http://mirror.suse.cz/install/SLP/${image_name}/${arch}/DVD1$device ";
```
Comment (mpagot) [inline tests/boot/boot_from_pxe.pm:129]:
```suggestion
            my $install_path;
            if (get_var('FORCE_INSTALL_FROM_MIRROR_HTTP')) {
                $install_path = 'install=' . get_required_var('MIRROR_HTTP') . "$device " ;
            } else {
                $install_path = "install=http://mirror.suse.cz/install/SLP/${image_name}/${arch}/DVD1$device ";
            }
```

Comment (mpagot) [reply tests/boot/boot_from_pxe.pm:129]: same result but using if/else

**Review by mpagot — APPROVED** [review]
LGTM

**Review by BillAnastasiadis — APPROVED** [review]
LGTM

**Pull request merged**